modules/ROOT/pages/scopes-and-collections-config.adoc

```diff
@@ -10,14 +10,7 @@ This section explains how to configure Scopes and Collections for Sync Gateway.
 It describes configuration options and provides example configuration code.
 
 == Understanding Buckets, Scopes, and Collections
-Each database can contain multiple buckets. 
-Each bucket can hold multiple scopes, and each scope can contain multiple collections.
-
-When you create a bucket, a bucket._default._default element is inherently included (see the documentation on xref:server:learn:data/scopes-and-collections.adoc#default-scope-and-collection[Default Scopes and Collections] in Couchbase Server). 
-Couchbase creates the default bucket to ensure backward compatibility.
-
-Only Couchbase Server (CBS) allows using the word 'default' in naming conventions. 
-As a result, you may see the term _default.default when creating a new bucket.
+You can define 1 custom scope per database with up to 1000 custom collections. If you don’t specify a custom scope and collection, any documents you create will be saved in the default scope and collection. For more information, examples and use cases, see https://www.couchbase.com/blog/scopes-collections-couchbase-mobile/[Scopes and Collections Support in Couchbase Mobile for Edge Applications].
```

Comment:
Formatting: New sentence on new line.

It looks like this compresses 3 paras into 1, and drops some of the information.
What's the intention here?